src/LineChart.js

```diff
@@ -17,32 +17,25 @@ export default class LineChart extends React.Component {
     data: PropTypes.array.isRequired,
     /**
      * Accessor function for line X values, called once per datum, or a single value to be used for the entire line.
-     */
-    x: CustomPropTypes.valueOrAccessor,
+     */ x: CustomPropTypes.valueOrAccessor,
```

Comment:
The formatting looks like it went rogue here and is repeated a few times below and in other files 😅

Reply:
I'm so confused because vscode should pick up the prettier config 😕